### PR TITLE
DHFPROD-9662: Disable Structured properties as a foreign key on the graph 

### PR DESCRIPTION
**Pull request by iveyMarklogic**
…raph

#### Checklist: 
```diff
- Note: do not change the below
```

- ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Code passes ESLint tests
- [x] Added Tests
- [x] Ran cypress tests on firefox locally
  

- ##### Reviewer:

- [N/A] Reviewed Tests
- [N/A] Added to Release Wiki

### CODE REVIEW

**Comment by ngodugu-marklogic** [comment]
The E2E test failure should be fixed as part of https://github.com/marklogic/marklogic-data-hub/pull/7345. We can merge this PR manually once we get the approvals @Sanjeevani19

**Review by briantang — APPROVED** [review]

**Comment by briantang** [comment]
This scenario is better tested manually - I updated the manual testing wiki for you @iveyMarklogic

**Pull request merged**
